bindings_ffi/src/mls.rs

```diff
@@ -144,6 +144,16 @@ impl FfiXmtpClient {
     pub fn installation_id(&self) -> Vec<u8> {
         self.inner_client.installation_public_key()
     }
+
+    pub fn release_db_connection(&self) {
+        self.inner_client.release_db_connection();
+    }
+
+    pub async fn db_reconnect(&self) -> Result<(), GenericError> {
+        let _ = self.inner_client.reconnect_db();
+
+        Ok(())
+    }
```

Comment:
Everything will error if they don't reconnect the database on foreground of the app. Should we be doing this reconnect for them if a release db connection gets called store a flag thats called `needs_reconnect` then when they try to do any other functions we reconnect for them? Or should we just let them manage the reconnection themselves? Is there some sort of place we could check that flag and do the reconnection for them easily?

Reply:
Since this is just for an iOS bug we could also only expose this in the iOS sdk limiting peoples access to these functions unless explicitly needed.

Reply:
+1 for hiding the complexity. If our users don't have a need for managing whether connections are active or not I think we should limit it's use as much as possible. Seems like a pretty huge footgun if someone mishandles it

Reply:
Is there a flag like delicate or dangerous API in Rust to discourage people from using it?